### DISCUSSION TITLE
How do I add files to shared/storage folder during deployment

### DISCUSSION THREAD

**Discussion by rabol** [deployphp/deployer#3606] [Help needed]
Hi

Once in a while I create som sample files that user of my website can download.

I store them in /storage/app/public, they are in my git repository but after deployment they are not available 'online'

I tried to add '/storage/app/public/' to the shared[], but that screwed up the whole deployment with this: error:

```
/var/www/xxxxxx/releases/178 && (mkdir -p bootstrap/cache storage storage/app storage/app/public storage/framework storage/framework/cache storage/framework/cache/data storage/framework/sessions storage/framework/views storage/logs)
[test] err mkdir: cannot stat ‘storage/app/public’: Too many levels of symbolic links
[test] exit code 1 (General error)
ERROR: Task deploy:writable failed!
task deploy:failed
```

**Comment by antonmedv** [comment]
It's a shared folder. Copy files in it after deploy:shared via additional task.
